### ISSUE TITLE
Format typo for option 46 leading to crash

### ISSUE THREAD

**Issue by olivierIllogika** [CyberShadow/dhcptest#16]
Got an exception when parsing option 46:

std.format.FormatException@/usr/local/opt/dmd/include/dlang/dmd/std/format.d(1257): Incorrect format specifier: %(%s, %s)

This change seems to fix it, but I've never touch D before.
```
		case OptionFormat.netbiosNodeType:
		enforce(bytes.length==1, "Bad netbiosNodeType data length");
-			f.writefln("%-(%s, %s)", bytes
+			f.writefln("%-(%s, %)", bytes
```

**Comment by CyberShadow** [comment]
Yes, that's the correct fix, thanks!

**Comment by CyberShadow** [comment]
This was fixed in 9f467b892739237c507aabc81eb8310f036aa15e.
